src/server/nuclearnet/fake_nuclearnet_client.ts

```diff
@@ -29,8 +36,9 @@ export class FakeNUClearNetClient implements NUClearNetClient {
   public connect(options: NUClearNetOptions): () => void {
     this.peer = {
       name: options.name,
-      address: '127.0.0.1',
-      port: 7447,
+      // Address and port are not used for anything yet, dummy values used.
+      address: 'fake_address',
+      port: 0,
```

Comment:
But what if you accidentally cast it to an int! Then it would drop the 0.1, clearly it needs to be 65536.1